CHANGELOG.md

```diff
@@ -1,10 +1,46 @@
 # Changelog
 
-## Unreleased
+## v0.88.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.88.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.88.0) and the [opentelemetry-collector-contrib v0.88.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.88.0) releases where appropriate.
 
 ### 🛑 Breaking changes 🛑
 
 - (Splunk) `smartagent`: Respect `JAVA_HOME` environment variable instead of enforcing bundle-relative value ([#3877](https://github.com/signalfx/splunk-otel-collector/pull/3877))
+- (Contrib) `k8sclusterreceiver`: Remove opencensus.resourcetype resource attribute ([#26487](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/26487))
+- (Contrib) `splunkhecexporter`: Remove `max_connections` configuration setting. ([#27610](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27610))
+  - use `max_idle_conns` or `max_idle_conns_per_host` instead.
+- (Contrib) `signalfxexporter`: Remove `max_connections` configuration setting. ([#27610](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27610))
+  - use `max_idle_conns` or `max_idle_conns_per_host` instead.
```

Comment:
```suggestion
  - use `max_idle_conns` or `max_idle_conns_per_host` instead.
- (Core) `exporterhelper`: make enqueue failures available for otel metrics ([#8673](https://github.com/open-telemetry/opentelemetry-collector/issues/8673)). This will prevent internal Collector `otelcol_exporter_enqueue_failed_<telemetry_type>` metrics from being reported unless greater than 0.

```